.github/workflows/on-main-push.yml

```diff
@@ -27,10 +28,10 @@ jobs:
           package-name: sfdx-git-delta
           token: ${{ secrets.RELEASE_PAT }}
 
-  release: 
+  release:
     needs: prepare-release
     runs-on: ubuntu-latest
-    if: ${{ needs.prepare-release.outputs.release_created == 'true' }}
+    if: needs.prepare-release.outputs.release_created == 'true'
```

Comment:
**thought:** I'm curious to see if it works

Reply:
actionlint detected this one, and the doc confirms :)
![image](https://user-images.githubusercontent.com/17500430/232287423-99eeb5f6-a06c-4f30-912a-ed490e0cccc4.png)